strongbox-security/strongbox-user-management/src/main/java/org/carlspring/strongbox/users/domain/AccessModelPathPrivileges.java

```diff
@@ -0,0 +1,57 @@
+package org.carlspring.strongbox.users.domain;
+
+import org.carlspring.strongbox.authorization.dto.PrivilegeDto;
+import org.carlspring.strongbox.users.dto.UserPathPrivilegesDto;
+
+import javax.annotation.concurrent.Immutable;
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.Set;
+
+import com.google.common.collect.ImmutableSet;
+import static java.util.stream.Collectors.toSet;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+@Immutable
+public class AccessModelPathPrivileges
+        implements Serializable
+{
+
+    private final String path;
+
+    private final boolean wildcard;
```

Comment:
@fuss86 @steve-todorov what means `widlcard` property?

Reply:
if `true` then accessed privileges are valid in the provided `path` and below the `path` tree (something like `/path/*`).
if `true` then accessed privileges are valid in the provided `path` without subtree

Reply:
@sbespalov `wildcard` is used in the UI to "tell" the backend to append `**` to the end of the `path` so that the path becomes `wildcard` and includes all subpaths.

Reply:
as I see backend uses same `wildcard` attribute without `**`
anyway, I think it will be good to have some javadocs for this to make it clearer for people